### PR TITLE
Kernel bug fixes

### LINKED ISSUE #91: Kernel parameter pti=on does nothing
The kernel parameter pti=on is do nothing because Page Table Isolation (PTI) is enabled by default in SEAPATH kernel (CONFIG_PAGE_TABLE_ISOLATION is set).

### LINKED ISSUE #94: Kernel parameter nohz_full is ignored
In order to use nohz_full kernel parameter, CONFIG_NO_HZ_FULL must be set.

Note: there is a bug in our kernel version with CONFIG_NO_HZ_FULL. So, to used it we must first update our kernel with the last RT patches revisions.

### PR DESCRIPTION
**Pull request by dupremathieu**
Fix the two kernel issues:
* fix #91
* fix #94

### CODE REVIEW

File: recipes-kernel/linux/linux-mainline-rt/defconfig [hunk 1]
```diff
@@ -223,9 +224,9 @@ CONFIG_IGC=y
 # CONFIG_NET_VENDOR_MARVELL is not set
 # CONFIG_NET_VENDOR_MICREL is not set
 # CONFIG_NET_VENDOR_MICROSEMI is not set
+# CONFIG_NET_VENDOR_NI is not set
```
Comment (eroussy) [inline recipes-kernel/linux/linux-mainline-rt/defconfig:227]: why this change ?

Comment (dupremathieu) [reply recipes-kernel/linux/linux-mainline-rt/defconfig:227]: I don't know, I've regenerated the defconfig. I guess it is because something has changed in the kernel or the defconfig was not generated correctly. Anyway, it has no impact at all at the end.

**Review by eroussy — APPROVED** [review]
